ldapauth.go

```diff
@@ -46,6 +46,8 @@ type Config struct {
 	CacheCookiePath            string   `json:"cacheCookiePath,omitempty" yaml:"cacheCookiePath,omitempty"`
 	CacheCookieSecure          bool     `json:"cacheCookieSecure,omitempty" yaml:"cacheCookieSecure,omitempty"`
 	CacheKey                   string   `json:"cacheKey,omitempty" yaml:"cacheKey,omitempty"`
+	CacheKeyLabel              string   `json:"cacheKeyLabel,omitempty" yaml:"cacheKeyLabel,omitempty"`
+	UseTLS                     bool     `json:"useTls,omitempty" yaml:"useTls,omitempty"`
```

Comment:
UseTLS was removed in the latest release

Reply:
@thpiron I recommend rebasing your branch with `main` to remove this.